### DISCUSSION TITLE
images for onboarding

### DISCUSSION THREAD

**Discussion by camrun91** [ApollosProject/apollos-templates#402] [General]
With moving onboarding to core is there a way we can pass images in now? @redreceipt

**Comment by redreceipt** [comment]
yep! override `BackgroundComponent` of `ui.Follow` (and others)

https://github.com/ApollosProject/apollos-apps/blob/145b8d45db2bc442267e9dd7a4a19262348b5784/packages/apollos-ui-onboarding/src/slides/Follow/Follow.js#L18

**Reply by camrun91** [reply]
I am still trying to figure out the overrides stuff but this is good. will mess with it and see what I can get.

**Reply by redreceipt** [reply]
in your `theme.js` file, something like this:

```js
const overrides = {
  "ui.Follow": {
    BackgroundComponent: () => () => <Image />
  }
}
```

The double function call is necessary for reasons that are hard to explain, I'm hoping to make it better soon.

**Comment by redreceipt** [comment]
@camrun91 is this resolved?

**Reply by camrun91** [reply]
yep
